site/src/sphinx/setup-configuration.rst

```diff
@@ -262,6 +263,16 @@ Core properties
 
     - the path of the management service. If not specified, the management service is mounted at ``/internal/management``.
 
+- ``zone`` (string)
```

Comment:
I do think `zone` may be confusing for users since it seems like a generic field which handles multi-zone deployment, but only affects plugin behavior - especially since there hasn't been a notion of `zone` in central dogma before (only internally).

What do you think of naming this more specifically if you don't have plans to expand the functionality of this field?

e.g.
`plugin-group-id`/`plugin-zone-id`

Alternatively, I think just using `groupId` of the server itself may be clearer - I think it is possible to deduce it from the peerId once a zk server starts.

Reply:
That's a good point. As you said, it's currently only used for plugins, but I didn't think the usage of it is limited to the plugins. It would also be useful to expose `zone` in the UI. Textual information is better than `groupId` which is a number.

Additionally, we may create repositories only in certain zones, as cloud providers do.

Reply:
By adding a coupling with zone, users won't be able to run plugins with arbitrary groups: i.e. run pluginA for ZoneA,ZoneB, run pluginB for ZoneC.

Having said this, I agree it's probably fine to proceed as-is since we haven't had requirements for this kind of use-case yet and we can always add a new Plugin type.

> Textual information is better than groupId which is a number.

Sounds fine, but when developing this field further I think it may still be useful if the relationship between `groupId` and `zone` is better defined. (Perhaps we can maintain an internal mapping of `zone <-> groupId` if only numbers are allowed)

Reply:
> Sounds fine, but when developing this field further I think it may still be useful if the relationship between groupId and zone is better defined. (Perhaps we can maintain an internal mapping of zone <-> groupId if only numbers are allowed)

I will think about this part further while working on mirroring. I need to change the configuration format to take all available zones to expose as a select box in the following PR.
For example:
```
zone: {
  currentZone: "ZoneA",
  allZones: ["ZoneA", "ZoneB", "ZoneC"]
}
```